Core/clim-core/frames/frames.lisp

```diff
@@ -901,7 +901,9 @@ frames and will not have focus.
 
```

Comment:
Please split this into multiple commits. Maybe header change, visual change, tracking-pointer change and rest.

Reply:
I'm not certain whether it makes sense, the file was simply rewritten to make further changes easier to understand.

Reply:
I'd rather squash these three commits, however I've put the header and visual change in two separate commits.

Regarding changing of protocols and the tracking-pointer change -- one depends on another and "splitting" them would rather entail writing a completely new implementation that is based on old protocols and I don't see how that would be an improvement.